garage/contrib/ros/docker/build.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
```

Comment:
Please remove this script, since it doesn't do anything more than a regular Docker command.